crypto/crypto/src/ed25519.rs

```diff
@@ -124,26 +128,7 @@ impl Ed25519Signature {
         if s == None {
```

Comment:
shouldn't the comment above s be beefed up to make it spectacularly clear `from_canonical_bytes` does the check we're looking for?